src/eqlpy/eqldjango.py

```diff
@@ -94,8 +98,86 @@ def _from_db_format(self, value):
         return json.loads(value)
 
 
+class EncryptedUniqueEquals(Lookup):
+    lookup_name = "eq"
+
+    def as_sql(self, compiler, connection):
+        lhs, lhs_params = self.process_lhs(compiler, connection)
+        rhs, rhs_params = self.process_rhs(compiler, connection)
+        # TODO: could this be done in get_prep_value?
+        # this could also be 'ore'?
+        rhs_params = [dict(e, q="unique") for e in rhs_params]
+        params = map(json.dumps, (lhs_params + rhs_params))
+        return "cs_unique_v1(%s) = cs_unique_v1(%s)" % (lhs, rhs), params
+
+
+EncryptedText.register_lookup(EncryptedUniqueEquals)
+
+
+class EncryptedTextContains(Lookup):
+    lookup_name = "contains"
```

Comment:
Can we call this `match` (instead of `contains`) to stay consistent with EQL?

Reply:
Sure. Done in the latest commit :)